setup.py

```diff
@@ -49,7 +49,7 @@
 ]
 
 setup(name='dark-matter',
-      version='1.0.9',
+      version='1.0.10',
```

Comment:
We're likely to get some users. Also, code elsewhere that calls `combineReads` with an `upperCase` argument can put the dark-matter version in its requirements to make sure it picks up a version that has the capability. Earlier (with no users and with us installing locally) it didn't matter at all. It's good to be in the habit of updating the version and pushing to PyPI.